src/virtual-source-buffer.js

```
@@ -718,17 +769,18 @@ export default class VirtualSourceBuffer extends videojs.EventTarget {
       });
 
       try {
-        destinationBuffer.updating = true;
         destinationBuffer.appendBuffer(tempBuffer);
       } catch (error) {
-        if (this.mediaSource_.player_) {
-          this.mediaSource_.player_.error({
-            code: -3,
-            type: 'APPEND_BUFFER_ERR',
-            message: error.message,
-            originalError: error
-          });
-        }
+        let currGOPStart = currentGOPStart(this.gopBuffer_,
+                                           this.mediaSource_.player_,
+                                           this.timeMapping_);
+
+        this.trigger({
```

Are there any other errors that might be triggered by `appendBuffer` that would be swallowed by this change? Some errors we want to trigger on the player, especially if they are "fatal" playback errors.

---

Seems like we should only trigger `bufferMaxed` if we see the `QuotaExceededError`?

---

I don't think it's swallowing any other errors, but I might as well gate it for safety as you suggest.